types/gulp-uglify/tsconfig.json

```diff
@@ -14,12 +14,7 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true,
-        "paths": {
```

Comment:
Why has this been removed?

Reply:
gulp-uglify uses uglify-js@3.x, so the mapping isn't needed so far as I can tell: https://github.com/terinjokes/gulp-uglify/blob/master/package.json#L14

Reply:
Afaik that is there to leave v2 maintained